email.php

```diff
@@ -165,6 +165,9 @@ protected function sendFormEmail($form, $params, $vars)
 
         // Send e-mail
         $this->email->send($message);
+
+        //fire event after eMail was sent
+        $this->grav->fireEvent('onEmailWasSent', new Event(['message' => $message, 'params' => $params, 'form' => $form]));
```

Comment:
For consistency, i think we should change this to `onEmailSent` . i know its close to `onEmailSend` but we don't use `was` anywhere in our event names.

Reply:
Done.
Hopefully I'm not coming up with any other events in the future, I'm getting the feeling we are running out of event names ;)